[2022-10-17 10:48:45,855] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.store_user backfill__2022-01-09T00:00:00+00:00 [queued]>
[2022-10-17 10:48:45,881] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.store_user backfill__2022-01-09T00:00:00+00:00 [queued]>
[2022-10-17 10:48:45,882] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:48:45,883] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-17 10:48:45,883] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:48:45,927] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_user> on 2022-01-09 00:00:00+00:00
[2022-10-17 10:48:45,950] {standard_task_runner.py:52} INFO - Started process 4925 to run task
[2022-10-17 10:48:45,982] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'store_user', 'backfill__2022-01-09T00:00:00+00:00', '--job-id', '181', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpgjhi3s5g', '--error-file', '/tmp/tmpuvidwn10']
[2022-10-17 10:48:45,997] {standard_task_runner.py:80} INFO - Job 181: Subtask store_user
[2022-10-17 10:48:46,140] {task_command.py:369} INFO - Running <TaskInstance: my_first_dag.store_user backfill__2022-01-09T00:00:00+00:00 [running]> on host 48b5b9061ccc
[2022-10-17 10:48:46,309] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-01-09T00:00:00+00:00
[2022-10-17 10:48:46,314] {postgres.py:127} INFO - Running copy expert: COPY users2 FROM stdin WITH DELIMITER AS ',', filename: /tmp/processed_user.csv
[2022-10-17 10:48:46,330] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-17 10:48:46,346] {python.py:173} INFO - Done. Returned value was: None
[2022-10-17 10:48:46,394] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=store_user, execution_date=20220109T000000, start_date=20221017T104845, end_date=20221017T104846
[2022-10-17 10:48:46,532] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-17 10:48:46,594] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
